react/features/chat/components/web/ChatMessage.js

```diff
@@ -31,7 +45,12 @@ class ChatMessage extends AbstractChatMessage<Props> {
 
         content.forEach(i => {
             if (typeof i === 'string') {
-                processedMessage.push(<Linkify key = { i }>{ i }</Linkify>);
+                if (this.isValidUrl(i)) {
+                    processedMessage.push(<Linkify key = { i }>{i}</Linkify>);
+                } else {
+                    processedMessage.push(<ReactMarkdown key = { i }>{ i }</ReactMarkdown>);
```

Comment:
Do we want to do this on mobile as well?

Reply:
Yes I'd like to implement this, however I think it would be better to implement this in a separate PR to keep the PRs as small and clean as possible. 

I'd like to hear your thoughts on this though